docs/architecture-decisions/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,43 @@
+# 1. Documentation: Record architecture decisions
```

Comment:
I can't say how much I value this. I've thought last night "I'll add the ADRs once the PR is in". Then I view it and you added it already.

Reply:
Gotchu fam! ✨